packages/i18n/locales/en/translation.json

```diff
@@ -549,6 +550,9 @@
     "updateProposalSubmissionConfigActionDescription": "Update the proposal submission paramaters for your DAO.",
     "updateVotingConfigActionDescription": "Update the voting parameters for your DAO.",
     "verifyingGovernanceToken": "Verifying CW20 token...",
+    "voteInSidebarLarge": "Vote from the sidebar on the right.",
+    "voteInSidebarResponsiveBody": "You can vote from the sidebar.",
+    "voteInSidebarResponsiveNudge": "Open it by pressing the profile circle in the bottom right or the button below.",
```

Comment:
Are we still doing this? I thought the voting was now not in the sidebar in responsive mode.